packages/flutter/test/widgets/value_listenable_builder_test.dart

```diff
@@ -18,6 +19,8 @@ void main() {
       child: ValueListenableBuilder<String?>(
         valueListenable: valueListenable,
         builder: (BuildContext context, String? value, Widget? child) {
+          ++rebuildCount;
```

Comment:
use `+= 1` here over `++`, see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer--over-